data/mods/No_Zombie_Animals/modinfo.json

```diff
@@ -0,0 +1,27 @@
+[
+{
+    "type": "MOD_INFO",
+    "mod-type": "SUPPLEMENTAL",
+    "ident": "No_Zombie_Animals",
+    "name": "No Zombie Animals",
+    "author": "Anonymous",
+    "description": "Removes zombie animals from the game.",
+    "path": "modinfo.json"
+},{
+    "type": "MONSTER_BLACKLIST",
+    "monsters": [
+       "mon_bear_smoky",
+       "mon_dog_zombie_rot",
+       "mon_dog_zombie_cop",
+       "mon_dog_skeleton",
+       "mon_zhark",
+       "mon_zombear",
+       "mon_zombie_dog",
+       "mon_zombie_mancroc",
```

Comment:
Hmm, I still feel uncertain about them since they are a kind of hybrid, but I guess with the way mutations work in CDDA they are probably more human than animal.